src/generators/java/JavaGenerator.ts

```diff
@@ -38,12 +36,14 @@ export class JavaGenerator extends AbstractGenerator {
   }
 
   async renderClass(model: CommonModel, inputModel: CommonInputModel): Promise<string> {
-    const renderer = new ClassRenderer(model, inputModel, this.options);
-    return renderer.render();
+    const presets = this.getPresets("class");
```

Comment:
Enum in which case? `this.getPresets` or `renderClass`? If first, then no, because `getPresets` is generic function and for example in GoLang we haven't class but struct 😄

Reply:
Was the first one I meant 👍